### ISSUE TITLE
Comments indicate deprecated `pixel{Width,Height}`, should these be warnings?

### ISSUE THREAD

**Issue by ntjess** [pyqtgraph/pyqtgraph#1962]
https://github.com/pyqtgraph/pyqtgraph/blob/0d8eba36a6a19ab6d8453f66a7dcf8d7e6f3c6a3/pyqtgraph/graphicsItems/GraphicsItem.py#L308-L330

↑ Should these functions use a `DeprecationWarning` instead, or are they not really deprecated? I assume `pixelLength` is replacing them?

**Comment by j9ac9k** [comment]
Yeah, I saw these comments and intentionally didn't add the warnings due to their usage in other parts of the library.  At the time I didn't notice equivalent functionality offered elsewhere, so I didn't add the warning to them; as I couldn't think of what to have those 

If there is a more robust equivalent functionality elsewhere in the library, I'm good with adding a warning and having that method in the warning message.

Wonder if this is something we should sort out after we get to the bottom of that InfiniteLineItem issue?
